feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go

```diff
@@ -63,6 +63,14 @@ var (
 			fn:   testJuniperECNConfig,
 		},
 	}
+	QoSNokiaEcnConfigTestcases = []Testcase{
+		{
+
+			name: "testNokiaECNConfig",
+			fn:   testNokiaECNConfig,
+		},
+	}
+	maxBurstSize = uint32(268435456)
```

Comment:
Why is maxBurstSize defined at the package level, instead of in the testcase function along with other parameters?

Reply:
Moved this function specific variable from package level to specific function.